Modules/System/Filter Tokens/README.md

```diff
@@ -11,6 +11,8 @@ You can add more filter tokens by subscribing to the following events:
 - `OnResolveDateTokenFromDateTimeFilter`
 - `OnResolveTimeTokenFromDateTimeFilter`
 
+> **Warning**  
+> Every filter token needs to start with a `%` sign as first charakter. Otherwise the events listed above might not get raised at all.
```

Comment:
"character" instead of "charakter".
"Might not" or "will not/won't"?

Reply:
Thanks! Will correct this tomorrow. But its actually "might". At least without making this sense much longer. 

From what I saw, date filter work without % and code text filter do not. 

Maybe someone from the server team can check and verify?
@JesperSchulz

Reply:
```suggestion
> Every filter token needs to start with a `%` sign as first character. Otherwise, the events listed above will not get raised at all.
```

Using _suggestions_ here is quite convenient.

Reply:
@mazhelez Next time I'll use suggestions for sure :) I just could not find where to add them. The 
![image](https://user-images.githubusercontent.com/34504100/215776407-ecb4f69b-5e93-42d6-8d31-749faea62023.png)
 symbol was unknown for me. The UI in DevOps is so much more intuitive for me when reviewing PRs ...